src/en/components/link/base.md

```diff
@@ -0,0 +1,22 @@
+---
+github: https://github.com/cds-snc/gcds-components/tree/main/packages/web/src/components/gcds-link
+figma: https://www.figma.com/file/mh2maMG2NBtk41k1O1UGHV/Canadian-Digital-Service%E2%80%A8---GC-Design-System?type=design&node-id=2646-8407&mode=design&t=Fc8G98RMn9FnseDi-0
+permalink: false
+tags: ['linkEN', 'header']
+---
+
+# Link <br>`<gcds-link>`
+
+_Also called: anchor, hyperlink, call to action._
+
+A link is a navigational element that brings a person to a new page, website, file, or section on the current page.
+
+{% docLinks locale stage figma github %}
+{% enddocLinks %}
+
+{% componentPreview "Link component preview" %}
+<gcds-link class="me-400" href="#">Link</gcds-link>
```

Comment:
Some of the feedback we received recently is that our examples tend to use the same text content for each element displayed, making it potentially confusing for users using AT. 

Can we add some context to each link (download link, mail to link, etc.)